playItAgain/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset = "UTF-8">
+    <title>Fun with WebAudio</title>
+    <script src='js/app.js'></script> 
+    <link rel="stylesheet" href="css/app.css">
+  </head>
+
+  <body>
```

Comment:
the formatting in this HTML file is far from par. Its very difficult to read what elements are inside of what other elements. The `<div>` on line 12 should cause you to intent line 13. Please run this through a formatter.